app/code/core/Mage/Adminhtml/Model/Observer.php

```diff
@@ -66,4 +66,22 @@ public function clearCacheConfigurationFilesAccessLevelVerification()
         Mage::app()->removeCache(Mage_Adminhtml_Block_Notification_Security::VERIFICATION_RESULT_CACHE_KEY);
         return $this;
     }
+
+    /**
+     * Set the admin's session lifetime based on config
+     */
+    public function cookieSetLifetime(Varien_Event_Observer $observer): void
```

Comment:
@justinbeaty did you choose this name for a specific reason or can we use something like setCookieLifetime?

Reply:
`setCookieLifetime` seems better. It's also in Mage/Customer/Model/Observer.php too. If you want to fix feel free, I am on the EAV branch right now.

Reply:
done!